src/PluginsManager/Microsoft.Performance.Toolkit.Plugins.Runtime/PluginsManagerException.cs

```diff
@@ -0,0 +1,281 @@
+﻿using System;
+using Microsoft.Performance.Toolkit.Plugins.Core;
+
+namespace Microsoft.Performance.Toolkit.Plugins.Runtime
```

Comment:
I think some of these classes should be `abstract` (e.g. `PluginsManagerException`)